### PR TITLE
Readme for rcp 4.27

### PR DESCRIPTION
**Pull request by Shee43**

### CODE REVIEW

**Comment by mickaelistria** [comment]
@sravanlakkimsetti isn't this file usually generated?

**Comment by sravanlakkimsetti** [comment]
yes this is generated @SarikaSinha Can you please take a look?

File: features/org.eclipse.rcp/rootfiles/readme/readme_eclipse.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1>
   <p>
     <a
       class="mozTocH1"
-      name="mozTocId299852">Release 4.26</a>
+      name="mozTocId299852">Release 4.27</a>
   </p>
   <p>
```
Comment (SarikaSinha) [inline features/org.eclipse.rcp/rootfiles/readme/readme_eclipse.html:24]: Last revised Nov 15 -> Needs to be updated.

**Comment by SarikaSinha** [comment]
2 more files needs to be added, look at https://github.com/eclipse-platform/www.eclipse.org-eclipse/pull/16/files
development/readme_eclipse_4.25.html
development/readme_eclipse_4.25.php

File: features/org.eclipse.rcp/rootfiles/readme/readme_eclipse.html (hunk 1 above)
Comment (SarikaSinha) [reply features/org.eclipse.rcp/rootfiles/readme/readme_eclipse.html:24]: The current file has Revised date as November, so either it has to be removed or updated based on the strategy adopted. 
The whole Read Me can be given a new look to have the least reference of versions etc.

**Review by SarikaSinha — APPROVED** [review]

**Pull request merged**